hack/test-go.sh

```diff
@@ -173,32 +213,12 @@ runTests() {
     if [[ $# -eq 0 ]]; then
       set -- $(kube::test::find_dirs)
     fi
-    kube::log::status "Running ${iterations} times"
-    fails=0
-    for arg; do
-      trap 'exit 1' SIGINT
-      pkg=${KUBE_GO_PACKAGE}/${arg}
-      kube::log::status "${pkg}"
-      # keep going, even if there are failures
-      pass=0
-      count=0
-      for i in $(seq 1 ${iterations}); do
-        if go test "${goflags[@]:+${goflags[@]}}" \
-            ${KUBE_RACE} ${KUBE_TIMEOUT} "${pkg}" \
-            "${testargs[@]:+${testargs[@]}}"; then
-          pass=$((pass + 1))
-        else
-          fails=$((fails + 1))
-        fi
-        count=$((count + 1))
-      done 2>&1
-      kube::log::status "${pass} / ${count} passed"
+    for p in $(seq 1 ${parallel}); do
+      runTestIterations ${p} "$@" &
     done
-    if [[ ${fails} -gt 0 ]]; then
-      return 1
-    else
-      return 0
-    fi
+    wait
```

Comment:
Since what we really want/need is "did any process fail"  So set a bash variable up in `runTestIterations()` on failure? (You could just use "fails" since you didn't make it local.)